storage/pkg/command/frontend.go

```diff
@@ -54,12 +54,32 @@ func Frontend(cfg *config.Config) *cli.Command {
 				desktopRedirectURIs[port] = fmt.Sprintf("http://localhost:%d", (port + 1024))
 			}
 
+			archivers := []map[string]interface{}{
+				{
+					"enabled":      true,
+					"version":      "2.0.0",
+					"formats":      []string{"tar", "zip"},
+					"archiver_url": "/archiver",
+				},
+			}
+
+			appproviders := []map[string]interface{}{
```

Comment:
```suggestion
			appProviders := []map[string]interface{}{
```

Line 75 suggests we're using camelCase here? If so, please also see line 82 ofc